### DISCUSSION TITLE
Override screensaver page and use different home page

### DISCUSSION THREAD

**Discussion by Bascht74** [Blackymas/NSPanel_HA_Blueprint#2049] [Q&A]
Hi, 

I am looking for a way to use the home page as screen saver and the entity page one as home page. 

So if there is no interaction with the display the home page will be shown with the TFT dimming down the brightness and if somebody is touching the display the brightness comes up and the first entity page is shown. 

Has somebody got this or something simmilar to work?

Thx for any hints...

**Comment by edwardtfn** [comment]
If you set "Timeout Sleep" to 0, your panel will never go to sleep page, so Home page will kind of be your sleep page.
By setting "Wake-up page" to Entity page x, you can set the page to be shown when your panel starts (and also when wake up, but if you set the previous suggestion, it will never sleep).
By setting "Timeout Page" to 0, your panel will never fall back to Home page automatically, so it will stay on the current page until you press the close button.

Does that helps?

**Comment by Bascht74** [comment]
Hi Edward, 
no it does not. 

I do want to have a "sleep" page, that should be the "home page". That is becaue it shows time and some other stuff I configured. 
But if I touch the panel I want it to go to button page one and use that as main navigation page. And then, after the timeout it should go back to the home page. 

With your suggestion the comlete active --> dim down --> sleep logic would not work nomore.

Right now I can set the WakeUp page:
<img width="355" alt="image" src="https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/66269310/a8c516aa-3e4f-4ce1-a4e6-7495504effab">

... but not "sleep page"

If you could be able to choose it, we could have a more flexible, e.g. use an entity page or the utility page as screensaver if you like (instead of the simple clock) ...
